models/topics-models.js

```diff
@@ -1,6 +1,13 @@
 const format = require("pg-format");
+const fs = require("node:fs/promises");
 const db = require("../db/connection");
 
+exports.readDoc = () => {
+  return fs.readFile(`${__dirname}/../endpoints.json`, "utf-8").then((data) => {
+    return JSON.parse(data);
```

Comment:
We don't need a model for this ticket. If you see my comment in the controller file, there's more info why 👍